common/djangoapps/util/models.py

```diff
@@ -12,3 +18,53 @@ class RateLimitConfiguration(ConfigurationModel):
     with the `can_disable_rate_limit` class decorator.
     """
     pass
+
+
+def uncompress_string(s):
+    """helper function to reverse django.utils.text.compress_string"""
+
+    try:
+        val = s.encode('utf').decode('base64')
+        zbuf = cStringIO.StringIO(val)
+        zfile = gzip.GzipFile(fileobj=zbuf)
+        ret = zfile.read()
+        zfile.close()
+    except:
```

Comment:
1. Please catch Exception
2. Please log an error if it happens, since that means we've saved corrupted stuff into the database.
3. Could you please explain in a comment the line `val = s.encode('utf').decode('base64')`?